docs/development/private-keys/package-signing.md

```diff
@@ -0,0 +1,169 @@
+# Package Signing Key Generation and Handling
+
+Last updated 2024-07-03.
+
+## Owernship
+Certificate and key generation and handling is overseen by ALESCo.
+
+## Key Generation
+
+This is a slow and tedious process.  It is important that no corners are cut and the utmost care is taken validating security through every step of the process.
```

Comment:
Worth adding an approximate reading / execution time?

Reply:
Probably not IMO.  This is mostly internal documentation that's just published for transparency-sake.